src/main/java/picard/illumina/ClusterDataToSamConverter.java

```diff
@@ -72,26 +71,56 @@
     // TODO: - https://github.com/broadinstitute/picard/issues/287
     // TODO: - HTS-spec issue: https://github.com/samtools/hts-specs/issues/109
     // TODO: - https://github.com/samtools/hts-specs/pull/119
-    private String molecularIndexTag             = "RX";
-    private String molecularIndexQualityTag      = "QX";
-    private final String molecularIndexDelimiter = "-";
-    private List<String> tagPerMolecularIndex    = Collections.emptyList();
+    // TODO: also add ~ and - to some htsjdk file and reference here.
+    private String molecularIndexTag = "RX";
+    private String molecularIndexQualityTag = "QX";
+    private static final String molecularIndexDelimiter = "-";
```

Comment:
Java convention for static final is uppercase with underscores, e.g. `MOLECULAR_INDEX_DELIMITER`